deploy/charts/csi-driver-spiffe/templates/deployment.yaml

```diff
@@ -37,9 +38,14 @@ spec:
           - --issuer-group={{ .Values.app.issuer.group }}
           - --trust-domain={{ .Values.app.trustDomain }}
 
-          - --leader-election-namespace={{ .Release.Namespace }}
+          - --leader-election-namespace=$(POD_NAMESPACE)
           - "--metrics-bind-address=:{{.Values.app.approver.metrics.port}}"
           - "--readiness-probe-bind-address=:{{.Values.app.approver.readinessProbe.port}}"
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
```

Comment:
question: What's the motivation for this change? I'm not saying we shouldn't do it - I just don't know why we _are_ doing it

Reply:
This is a very subjective change, it makes it so our operator manifests have less namespaces that have to be updated when we change the namespace. This change is copied from the changes we had in that repo.